### ISSUE TITLE
<expression> or FirebaseRulesTokenType.) expected, got '['

### ISSUE THREAD

**Issue by serinuntius** [anboralabs/intellij-firebase-highlighter#16]
Hello!

Thanks for the awesome plugin!
By the way I ran into a bug. I wish I could fix it myself, but... 
In the meantime, I'll report back to you.

## Environment
Version: 1.1.1-SNAPSHOT

## Result
<expression> or FirebaseRulesTokenType.) expected, got '['

![image](https://user-images.githubusercontent.com/10571219/97849942-ad6d6800-1d36-11eb-89c2-db326b58f457.png)

```
rules_version = '2';

service cloud.firestore {
    match /databases/{database}/documents {
        function documentRef(paths) {
            return path([['databases', database, 'documents'].join('/'), paths.join('/')].join('/'));
        }
    }
}

```

**Comment by dalgarins** [comment]
Thanks for testing the plugin, the changes gonna be available for version: 1.1.2
![Screen Shot 2020-11-02 at 1 08 54 PM](https://user-images.githubusercontent.com/8527174/97903214-ad587280-1d0c-11eb-8e62-387072fda684.png)

**Comment by serinuntius** [comment]
The fix is too soon! Thank you!
